rules/windows/process_creation/proc_creation_win_reg_service_imagepath_change.yml

```
@@ -1,4 +1,4 @@
-title: Service ImagePath Change with Reg.exe
+title: Suspicious Service ImagePath Value Change Via Reg.EXE
```

```suggestion
title: Changing Existing Service ImagePath Value Via Reg.EXE
```
the image path itself is not suspicious, so I would rephrase the title